x-pack/plugins/apm/scripts/kibana-security/setup-custom-kibana-user-role.ts

```diff
@@ -6,27 +6,14 @@
 
 /* eslint-disable no-console */
 
-import yaml from 'js-yaml';
 import axios, { AxiosRequestConfig, AxiosError } from 'axios';
-import fs from 'fs';
 import { union, difference, once } from 'lodash';
-import path from 'path';
 import { argv } from 'yargs';
 
-const config = yaml.safeLoad(
-  fs.readFileSync(
-    path.join(__filename, '../../../../../../config/kibana.dev.yml'),
-    'utf8'
-  )
-);
```

Comment:
Removed the magic around reading creds from the existing kibana.yml file. Now everything must be passed in as cli args. A little more tedious but a lot more clear what happens